src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -965,6 +965,17 @@ def test_git_layers(self):
         for config in report['configurations']:
             self.validate_cfs_config(config['name'])
 
+    @skip_test_if_csm_var_missing(['branch_name', 'commit_hash'])
+    def test_no_resolve_branches(self):
+        """Test creating multiple CFS configurations with git-based layers with --no-resolve-branches"""
+        result = self.run_bootprep('git-layers-config.yaml', '--format json', '--no-resolve-branches')
```

Comment:
All the bootprep options are supposed to be in a single string in the second argument of this function. The third argument is `check`, which does not need to be changed from the default.

```suggestion
        result = self.run_bootprep('git-layers-config.yaml', '--format json --no-resolve-branches')
```